### PR TITLE
Update CTA

### PR DESCRIPTION
**Pull request by colbyallenc**
- Remove repetition of this line: “Design systems are intended to cross complex boundaries, but realizing the full value of your investment is tricky work.” 
- Add's correct line in the repeated lines place: "What do you need to support your design system?
- CTA buttons say “Take the Assessment”

### CODE REVIEW

File: src/sections/2021/section-5.js [hunk 1]
```diff
@@ -144,9 +144,9 @@ const Section5 = () => (
         </svg>
 
         <h3>Design systems are intended to cross complex boundaries, but realizing the full value of your investment is tricky work.</h3>
-        <p>Design systems are intended to cross complex boundaries, but realizing the full value of your investment is tricky work.</p>
+        <p>What do you need to support your design system?</p>
         <p>Take the Maturity Model Assessment to get feedback on where you are now, and suggestions that will help you move forward to benefit your whole organization.</p>
-        <a href="https://example.com" className="cmp-button cmp-button--inverse">Take the Survey</a>
+        <a href="https://example.com" className="cmp-button cmp-button--inverse">Take the <strong>Assessment</strong></a>
```
Comment (zastrow) [inline src/sections/2021/section-5.js:149]: Assessment shouldn’t be bolded here, please remove `<strong>` tags.

File: src/sections/2021/section-5.js [hunk 2]
```diff
@@ -144,9 +144,9 @@ const Section5 = () => (
         </svg>
 
         <h3>Design systems are intended to cross complex boundaries, but realizing the full value of your investment is tricky work.</h3>
-        <p>Design systems are intended to cross complex boundaries, but realizing the full value of your investment is tricky work.</p>
+        <p>What do you need to support your design system?</p>
```
Comment (zastrow) [inline src/sections/2021/section-5.js:147]: I did not see this in the content edits doc. Was this from a separate conversation?

**Review by zastrow — CHANGES_REQUESTED** [review]
One change, one question.

File: src/sections/2021/section-5.js (hunk 1 above)
Comment (colbyallenc) [reply src/sections/2021/section-5.js:149]: ok! I assumed the bold on the doc meant to make the text bold. 
Screen Shot 2021-07-08 at 9 34 17 AM

File: src/sections/2021/section-5.js (hunk 2 above)
Comment (colbyallenc) [reply src/sections/2021/section-5.js:147]: This is how it is written in the Content Edits Doc. Should I remove this line?

Screen Shot 2021-07-08 at 9 34 55 AM

Comment (zastrow) [reply src/sections/2021/section-5.js:147]: Ah. That’s in the [main doc](https://docs.google.com/document/d/1GZoY2jo4G8bYp9qW1ja8qxHgwiza4zO73El6Zsey2lk/edit#), not in the [edit doc](https://docs.google.com/document/d/1AvTqkMunkK0f1I-iCD6O0ZK9WNKA8gAp4QNt8lPv9yU/edit). That needs to be verified with Kaitlyn.

File: src/sections/2021/section-5.js (hunk 1 above)
Comment (zastrow) [reply src/sections/2021/section-5.js:149]: Makes sense why you took it that way. That was to emphasize which word needed changed.

**Pull request merged**